optlib/terraform.ctags

```diff
@@ -25,11 +25,11 @@
 --kinddef-terraform=p,Provider,Terraform Provider
 --kinddef-terraform=m,Module,Terraform Module
 --kinddef-terraform=o,Output,Terraform Output
---regex-terraform=/^[[:space:]]*resource[[:space:]]*"([^"]*)"[[:space:]]*"([^"]*)"/\2/r,Resource/
---regex-terraform=/^[[:space:]]*data[[:space:]]*"([^"]*)"[[:space:]]*"([^"]*)"/\2/d,Data/
---regex-terraform=/^[[:space:]]*variable[[:space:]]*"([^"]*)"/\1/v,Variable/
---regex-terraform=/^[[:space:]]*provider[[:space:]]*"([^"]*)"/\1/p,Provider/
---regex-terraform=/^[[:space:]]*module[[:space:]]*"([^"]*)"/\1/m,Module/
---regex-terraform=/^[[:space:]]*output[[:space:]]*"([^"]*)"/\1/o,Output/
+--regex-terraform=/^resource[[:space:]]"([^"]+)"[[:space:]]"([^"]+)"/\2/r,Resource/
```

Comment:
You use both kind letter and kind long name in "r,Resource".
However, `r` is already defined with `--kinddef-terraform`. So you should use only 'r' here.
```
--regex-terraform=/^resource[[:space:]]"([^"]+)"[[:space:]]"([^"]+)"/\2/r/
```